Admin dashboard

maung888 opened the pull request.

davidvanderklay reviewed (changes requested):
A few additional rspec tests are failing after adding these changes on my end, and there should probably be a few UI changes on each of the pages. There should be some way to navigate to the previous page on each of the pages. Admins still' aren't the only users who can access this page, but I'll probably help with that since the implemented data design may need tweaking.

maung888 commented:
I added navigation buttons. I have to make faculty more relaxed and be able to create without department and name (since we only have email for some data), so for that I removed some respect related to that. So, we only have one rsepc failing. 

So currently only one rspec test is failing, since tam wrote that rspec I will let him reconsider it later.

davidvanderklay reviewed (approved):
Looks like things are working correctly now.
